app/controllers/api/storage.php

```diff
@@ -171,6 +172,8 @@
             throw new Exception('Bucket not found', 404);
         }
 
+        $read??=$bucket->getAttribute('$read', []); // By default inherit read permissions
```

Comment:
```suggestion
        $read ??= $bucket->getAttribute('$read', []); // By default inherit read permissions
```

I would leave some space for readabilty.